### ISSUE TITLE
pipx instructions don't install the latest

### ISSUE THREAD

**Issue by cponder** [vlaci/openconnect-sso#125]
I ran these instructions
```
sudo apt-get -y install pipx
pipx install "openconnect-sso"
```
This gave the output
```
  installed package openconnect-sso 0.7.3, installed using Python 3.10.6
  These apps are now globally available
    - openconnect-sso
done! ✨ 🌟 ✨
```
So it looks like it installed the backlevel 0.7.3 instead of the more current 0.8.0.
How can I get the latest?

**Comment by cponder** [comment]
Also, when I try to run this backlevel version I get this error
```
ModuleNotFoundError: No module named 'PyQt5'
```
Is there a simple command to install this? These looked like they worked
```
sudo apt-get -y install python3-pyqt5
pip install pyqt5
```
but the errors still happen. I'm running Ubuntu 22.04.

**Comment by justinrporter** [comment]
I have the same issue installing on MacOS Ventura (13.2) with a virtualenv.

**Comment by Xandir150** [comment]
1. try `pipx install pyqt5`

**Comment by rapgro** [comment]
see also #127

**Comment by Xandir150** [comment]
ARM MacOS
also can use next steps:
```
pipx install openconnect-sso
/Users/<USERNAME>/.local/pipx/venvs/openconnect-sso/bin/python -m pip install pyqt5 --config-settings --confirm-license= --verbose
```

**Comment by bassdr** [comment]
latest version 0.8.0 seem to be called [openconnect-ssom1](https://pypi.org/project/openconnect-ssom1/) in pipx, no clue what the `m1` stands for, or if they 0.7.3 as more stable up there?

**Comment by dagbdagb** [comment]
> latest version 0.8.0 seem to be called [openconnect-ssom1](https://pypi.org/project/openconnect-ssom1/) in pipx, no clue what the `m1` stands for, or if they consider 0.7.3 as more stable up there?

Great. Managed to get it to play? That 'm1' appears to cause issues. What is the right symlinking/renaming/whatever to get it to work?

**Comment by bassdr** [comment]
> > latest version 0.8.0 seem to be called [openconnect-ssom1](https://pypi.org/project/openconnect-ssom1/) in pipx, no clue what the `m1` stands for, or if they consider 0.7.3 as more stable up there?
> 
> Great. Managed to get it to play? That 'm1' appears to cause issues. What is the right symlinking/renaming/whatever to get it to work?

Nope, no luck either, `pipx install openconnect-ssom1` is broken for me too. The site-package folder will be `openconnect_ssom1` but then it will later refer to `openconnect-sso` which does not exist. I did not bother hacking filesystem with symlinks and instead compiled it from this repo instead, and it does seem to work so far.

This might not be for the faint-hearted tho, and TBH I don't think there is much in this version that worth the effort.
https://github.com/vlaci/openconnect-sso/compare/v0.7.3...v0.8.0

> ## v0.8.0
>
>- Updating dependencies
>- Add support for MFA popup dialogs
>- Pass cookie in STDIN to hide its value in `ps` output
>- Support NixOS 21.11

If you are interrested, I'm on Gentoo and I created ebuild files in my personal overlay in order to install openconnect-sso from Gentoo's package manager: https://github.com/bassdr/gentoo-local-overlay/tree/master/dev-python/openconnect-sso

**Comment by socketbox** [comment]
I used `pipx install --force git+https://github.com/vlaci/openconnect-sso@428ab94`

**Comment by vlaci** [comment]
I have uploaded v0.8.1 to pypy containing all the changes from current git revision.
I have still no ability to test this package, so any feedback is wellcome
